apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -87,56 +81,23 @@ function TeacherDashboard({
     }
   });
 
-  const [teacherNavV2Enabled, setTeacherNavV2Enabled] = useState(false);
-
-  useEffect(() => {
-    setTeacherNavV2Enabled(
-      DCDO.get('teacher-local-nav-v2', false) ||
```

Comment:
It looks like you got rid of the DCDO flag altogether here.  Was that your intention?

Reply:
Yes, we are instead relying on [this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/studio/pages/teacher_dashboard/show.js#L101) use of the flag that was added previously